ensembl_metadata/models/genome.py

```diff
@@ -0,0 +1,160 @@
+
+
+class GenomeFeature(models.Model):
+    genome_feature_id = models.AutoField(primary_key=True)
+    genome = models.ForeignKey(Genome, on_delete=models.CASCADE,
+                               related_name='features')
+    type = models.CharField(max_length=32)
+    analysis = models.CharField(max_length=128)
+    count = models.IntegerField()
+    genome_database = models.ForeignKey(GenomeDatabase, models.CASCADE,
+                                        related_name='features')
+
+    class Meta:
+        unique_together = (('genome', 'type', 'analysis', 'genome_database'),)
+
+
+class GenomeRelease(models.Model):
+    genome_release_id = models.AutoField(primary_key=True)
+    genome_uuid = models.ForeignKey(Genome, on_delete=models.CASCADE,
+                                    related_name='releases',
+                                    to_field='genome_uuid')
+    division = models.ForeignKey(Division, on_delete=models.CASCADE,
+                                 related_name='releases')
+    data_release = models.ForeignKey(DataRelease, on_delete=models.CASCADE,
+                                     related_name='releases')
+
+
+class GenomeVariation(models.Model):
+    genome_variation_id = models.AutoField(primary_key=True)
+    genome = models.ForeignKey(Genome, on_delete=models.CASCADE,
+                               related_name='variations')
+    type = models.CharField(max_length=32)
+    name = models.CharField(max_length=128)
+    count = models.IntegerField()
+    genome_database = models.ForeignKey(GenomeDatabase, on_delete=models.CASCADE,
+                                        related_name='variations')
+
+    class Meta:
+        unique_together = (('genome', 'type', 'name', 'genome_database'),)
+
+
+class ComparaAnalysis(models.Model):
+    compara_analysis_id = models.AutoField(primary_key=True)
+    genome_compara_analysis = models.ManyToManyField('ensembl_metadata.Genome',
+                                                     through='GenomeComparaAnalysis')
+    method = models.CharField(max_length=50)
+    set_name = models.CharField(max_length=128, blank=True, null=True)
+    dbname = models.CharField(max_length=64)
+    data_release = models.ForeignKey(DataRelease, on_delete=models.CASCADE,
+                                     related_name='analyses')
+    division = models.ForeignKey(Division, on_delete=models.CASCADE,
+                                 related_name='analyses')
+
+    class Meta:
+        unique_together = (('division', 'method', 'set_name', 'dbname'),)
+
+
+class ComparaAnalysisEvent(models.Model):
+    compara_analysis_event_id = models.AutoField(primary_key=True)
+    compara_analysis = models.ForeignKey('ComparaAnalysis', on_delete=models.CASCADE,
+                                         related_name='events')
+    type = models.CharField(max_length=32)
+    source = models.CharField(max_length=128, blank=True, null=True)
+    creation_time = models.DateTimeField()
```

Comment:
For this kind of field, best is to use https://docs.djangoproject.com/en/3.1/ref/models/fields/#django.db.models.DateField.auto_now , this way it's automatically populated by Django.